README.md

```diff
@@ -1,17 +1,17 @@
 # AI Ecosystem Extension Specification
 
+This document explains the AI Ecosystem Extension to the
+[Field Boundaries for Agriculture (fiboa) Specification](https://github.com/fiboa/specification).
+
 - **Title:** AI ecosystem
 - **Identifier:** <https://fiboa.github.io/ai-ecosystem/v0.1.0/schema.yaml>
 - **Property Name Prefix:** ml
 - **Extension Maturity Classification:** Proposal
 - **Owner**: @hannah-rae @subash-khanal @yanglexie @jacobsn @eddiechoi00
 
-This document explains the AI Ecosystem Extension to the
-[Field Boundaries for Agriculture (fiboa) Specification](https://github.com/fiboa/specification).
-
-This extension adds support for properties useful for integrating field boundary data with the AI/ML ecosystem.
+This extension also adds support for properties useful for integrating field boundary data with the AI/ML ecosystem.
 
-- Examples: (TODO)
```

Comment:
These examples are actually TODO and should be marked as such (that's why the CI is failing)

Reply:
Hi @m-mohr I updated the PR to include the TODO but it is still failing checks. Apologies for if I am missing something silly. Thanks so much for your help.

Reply:
Sorry, I was not very clear. The examples are a todo and that's why CI is failing. It's not failing because the text is missing, but because the examples are invalid.